source/install/upgrade_from_mongodb.rst

```diff
@@ -17,38 +18,46 @@ An in-place upgrade is done with existing data in the server.  Generally
 speaking, this is stopping the server, removing the old packages, installing the
 new server and starting it with the same data files.  While an in-place upgrade
 may not be suitable for high-complexity environments, it should work in most
-cases.
+cases.  
 
 .. warning::
 
-   Before starting the upgrade process it is recommended to perform a full
+   Before starting the upgrade service, we recommend to perform a full
    backup of your data.
 
-The upgrade process depends on the distribution you are using:
+The sections below describe an in-place upgrade of a standalone instance and a replica set without encryption. 
 
 .. contents::
    :local:
 
-Upgrading on Debian or Ubuntu
+.. _upgrade_standalone:   
+
+Upgrading a standalone instance or a single-node replica set 
 ================================================================================
 
+The upgrade service depends on the distribution you are using:
```

Comment:
upgrade procedure?

Reply:
"upgrade procedure" is best. "upgrade process" or "upgrade steps" would also be OK.